tidb-cloud/serverless-export.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Export Data from TiDB Serverless
+summary: Learn how to export data from TiDB Serverless clusters.
+---
+
+# Export Data from TiDB Serverless
+
+TiDB Serverless Export (Beta) is a service that enables you to export data from a TiDB Serverless cluster to local storage or an external storage service. You can use the exported data for backup, migration, data analysis, or other purposes.
+
+While you can also export data using tools such as [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) and TiDB [Dumpling](https://docs.pingcap.com/tidb/dev/dumpling-overview), TiDB Serverless Export offers a more convenient and efficient way to export data from a TiDB Serverless cluster. It brings the following benefits:
+
+- Convenience: the export service provides a simple and easy-to-use way to export data from a TiDB Serverless cluster, eliminating the need for additional tools or resources.
```

Comment:
One more advantage is you don't have to take care of memory allcoation. For mysqldump and dumpling, it's easy to exceed memory limitation, especially for free tier the limitation is only 256MiB.